easybuild/easyconfigs/m/MuTect/MuTect-1.1.7-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Version 1.1.7 modified by
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'MuTect'
+version = '1.1.7'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://www.broadinstitute.org/cancer/cga/mutect'
+description = """ MuTect is a method developed at the Broad Institute for the reliable
+ and accurate identification of somatic point mutations in next generation sequencing
+ data of cancer genomes. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# Need to be logged in to download, at this URL
+# source_urls = ['http://www.broadinstitute.org/cancer/cga/sites/default/files/data/tools/mutect/']
```

Comment:
this URL sends you back to a landing page, just point to http://www.broadinstitute.org/cancer/cga/mutect_download instead?

```python
# download requires registration, see http://www.broadinstitute.org/cancer/cga/mutect_download
sources = ...
```